composer.json

```diff
@@ -24,5 +24,10 @@
         "psr-4": {
             "lewiscowles\\Rfc\\": "src"
         }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "lewiscowles\\Rfc\\": "tests"
```

Comment:
As in other repo tests really need their own namespace, they are not part of the body of code.

Reply:
I will separate the different namespace in the next PR.